src/pcapi/notifications/push/transactional_notifications.py

```diff
@@ -40,3 +42,19 @@ def get_bookings_cancellation_notification_data(booking_ids: List[int]) -> Optio
             body=f"""Ta {cancelled_object} "{bookings[0].stock.offer.name}" a été annulée par l'offreur.""",
         ),
     )
+
+
+def get_tomorrow_booking_notification_data(booking_id: int) -> Optional[TransactionalNotificationData]:
+    booking = Booking.query.filter_by(id=booking_id).join(Booking.stock).join(Stock.offer).one()
+
+    if not booking or booking.isCancelled:
```

Comment:
```suggestion
    if  booking.isCancelled:
```

puisque désormais, `booking` ne peut plus être `None`. Soit on en a un, soit `one()` pus haut lèvera une erreur.